### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by tosh-21**
Word Check working
Game Functionality working
Print Final Array of Arrays not working

### CODE REVIEW

File: wordle/scratch_3.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+}
+
+func AppendSlice(source [][]string, value []string) [][]string {
+	return append(source, value)
+}
+
+func PrintArray(array [][]string){
+	for i := 0; i < len(array); i++{
+		for j := 0; j <= len(array); j++{
```
Comment (helmus) [inline wordle/scratch_3.go:28]: Length of which array ?

**Review by helmus — CHANGES_REQUESTED** [review]

File: wordle/scratch_3.go (hunk 1 above)
Comment (tosh-21) [reply wordle/scratch_3.go:28]: This is incorrect implementation. The PrintArray2 function below this is the one that is correct.

Comment (helmus) [reply wordle/scratch_3.go:28]: Why is this included in the PR ?

Comment (tosh-21) [reply wordle/scratch_3.go:28]: Mistake. I'll overwrite it.

Comment (tosh-21) [reply wordle/scratch_3.go:28]: I don't like it

**Pull request closed without merging**